.github/workflows/pipeline_run_from_notebook.yaml

```diff
@@ -2,18 +2,16 @@ name: Create Pipeline Run from Kubeflow Notebook
 on:
   pull_request:
     paths:
+    - tests/gh-actions/install_KinD_create_KinD_cluster_install_kustomize.sh
     - .github/workflows/pipeline_run_from_notebook.yaml
     - apps/jupyter/notebook-controller/upstream/**
     - apps/pipeline/upstream/**
-    - tests/gh-actions/kind-cluster.yaml
-    - tests/gh-actions/install_kind.sh
-    - tests/gh-actions/install_kustomize.sh
     - tests/gh-actions/install_istio.sh
     - tests/gh-actions/install_cert_manager.sh
     - common/cert-manager/**
-    - common/oauth2-proxy/**
+    - common/oidc-client/oauth2-proxy/**
```

Comment:
Do we have common/oidc-client/oauth2-proxy that's replacing common/oauth2-proxy in the paths here?

Reply:
We have common/oauth2-proxy

Reply:
Then, was this a suggested change made here and in other files.

Reply:
No worries, I fixed the typo in my new PR.